docs/content/test_runner/Metrics.md

```diff
@@ -4,7 +4,7 @@
 
 Metrics available at:
 
-`$namespace.testrunner.$buildId.$instrumentationConfigName.`
+`$namespace.testrunner.$buildId.$projectName.$instrumentationConfigName.`
```

Comment:
How about project path? Name is not unique

Reply:
it will be much more symbols in metric, and names are unique for now;
but it could lead to hard-to-debug problems
i don't know how what is better here